### DISCUSSION TITLE
大家冷氣都開幾度呢

### DISCUSSION THREAD

**Discussion by shuxian12** [1101-GS4719-C/Python-Community-GS4719-C#1394] [General]
如題最近天氣越來越熱，有許多新聞也都開始討論冷氣應該要開幾度好呢？？

我自己從小就是靠一台電風扇打天下，連冷氣都沒開，走一個清淨自然涼的路線，但好像因為地球暖化越來越嚴重，真的到了需要開冷氣的地步，所以我家老爹終於願意在我房間換新的冷氣，但我自己覺得26度吹下去，感覺好像就會被冷醒，自己比較喜歡開27度，然後再搭配電風扇，但終點是電風扇不能對著我吹，不然也會冷醒

不知道大家都喜歡開幾度呢？？？？

**Comment by juliaouo** [comment]
我會開28度，因為我本身是滿怕冷的人，但是學校宿舍的冷氣不知道是啥問題，開26度溫度計量室內室溫是22度，這樣是可以報修的嗎？另外我喜歡用那種會轉的電風扇，比較不會著涼。

**Reply by shuxian12** [reply]
我自己是覺得如果戶外只有20 21度開冷氣開27度，就會覺得房間滿熱的，但如果外面有25 26度，開27度又會覺得太冷，都不知道學校的冷氣機在幹嘛

**Comment by TW-Maverick** [comment]
我在家都是睡覺時才開冷氣，大概也都開27~28左右，但是一定要配電風扇，不然冷的速度會非常的慢 ，
在宿舍時我通常是熱到不行才會開冷氣，因為我蠻怕冷的，而且重點是還要多一筆花費🤣，
但很多時候我會配合室友吹冷氣，不然他們會很emo！

**Comment by Superlan8** [comment]
現在全球暖化太嚴重了，我認為少了冷氣夏天根本活不下去，而25-26度是我平時會開的溫度，我強力推薦一定要配電風扇，配電風扇幫助房間空氣循環，加速降溫，也會減少冷氣壓縮機運轉的時間。再補充一點，大家可能平時會忘記的，要定時清理冷氣濾網，這也是可以幫助冷氣更涼的原因之一。

**Comment by shuxian12** [comment]
但我聽說很多男生都喜歡開19、20度，感覺就超級冷，但他們好像有時候依舊覺得很熱
自從家裡換了變頻的冷氣機之後都不需要擔心電費的問題了，但就是變頻冷氣機好貴( ･᷄ὢ･᷅ )

**Comment by pigeon9975** [comment]
學校的冷氣真的超有問題！！！！明明遙控器上寫26度，但過一段時間後冷氣就會自動跳成22度，而且怎麼按都沒辦法升溫，害我每天早上起來都像置身於冷凍庫，感覺好浪費電，對身體也不太好，順便在這邊問有人知道怎麼解決嗎QQ

**Comment by sjh501** [comment]
我是從小一台電扇一家人一起用，那個時候28度對我來說就算涼爽，而且只要進冷氣房吹到風就會開始頭痛。在國中後開始有冷氣都開26到27度左右，到現在讀大學好像越來越怕熱，28度就會覺得沒開冷氣待不下去，尤其是在上課或讀書的時候。但是如果有風一直吹雖然會覺得熱，頭痛的毛病還是沒有什麼改變，要吃藥才會舒坦一些。

**Comment by quan0715** [comment]
人常說心靜自然涼，你想想古代沒有冷氣的時候大家不也過得好好的，總歸就是都市熱島效應太嚴重，大家應該多多省電，採用意志力降溫法，對你我都好。

**Comment by zephyrxin01** [comment]
我覺得宿舍的冷氣機不太客觀...開了27度還是超冷，但是關掉又會超級悶，開了電風扇也沒有用。看著宿舍電表嘆氣，夏天一到一天就噴了差不多40塊錢QQ

**Comment by Alexgns** [comment]
目前晚上大概都開20度左右，但太熱的話可能會睡不著，早上起床會很冷，平常大約23-25度，平常沒在關冷氣感覺電費耗蠻兇的

**Comment by alstinlin** [comment]
大一住宿幾乎都會開著，但搬出去住之後除非太熱，不然基本上不太會開，因為電費太貴了qq，不過開冷氣還有一個好處，就是可以順便除濕，這點我認為還不錯~

**Comment by 1104030360** [comment]
我以前都開23度後來改成開電風扇（除非太熱），因為這樣比較環保愛地球，也省電省錢！

**Comment by JessyCheng2003** [comment]
我覺得開22～23度是最舒適的溫度，開冷氣除了能降溫還可以除濕，所以有時候很潮濕的時候我也會開冷氣。

**Comment by pudding0803** [comment]
我在家都會盡量不開冷氣，但如果真的受不了的話，我都會先調超低（20 左右），等到覺得溫度降下來之後再往上調（只是為了快速降溫？），雖然不知道這樣會不會有什麼問題，如果有知道的再麻煩跟我說一下🥲，謝謝！

不過我覺得最麻煩的是多個人在同一個空間開冷氣，因為每個人對溫度的感受不同，所以可能常常會一下有人覺得熱、一下有人覺得冷。像我自己就比較怕熱一點，天氣冷的時候沒差，可是熱的時候別人不一定覺得熱😭

**Comment by ALAN30359704** [comment]
我都開除濕23度，但因爲除濕都讓我覺得特別冷（？？所以就常常不小心吹到感冒，最近上課常常聽到有人上課流鼻水，請大家多多注意。

**Comment by rayyichou** [comment]
我在家都吹27度是我覺得剛剛好的溫度，如果能不開冷氣我就盡量不開冷氣，畢竟冷氣會製造更多的熱能排放到外面，雖然大家開玩笑說北極熊吹很久，換我們吹了。但還是要保護環境。

**Comment by youtinglin** [comment]
如果以住學校宿舍的話，夏天其實我都盡量在公共空間吹冷氣，回宿舍也只有睡覺的時候才開冷氣，不僅環保也省錢。我記得之前台電有公告說冷氣每調高一度，可節省空調電力6%，且溫度設在26至28度加上電扇，可以有效節省電費，因此我都開26度，而且其實中央大學位在台灣北部，相比南部學校，大概也只有5月到9月需要吹到冷氣，其他時候吹電風扇就蠻涼了，但也可能是我比較怕冷的關係。

**Comment by aa103967622** [comment]
大部分時間都是開25到26度，我覺得在這個溫度範圍最舒適，尤其是在晚上睡覺的時候，開著這個溫度可以保證我快速入眠，但是！只要溫度不小心開低個1到2度，就會感覺冷到爆炸⋯⋯半夜會冷醒的那種

**Comment by 111607004** [comment]
我覺得宿舍的冷氣溫度真的是裝飾用的，明明調了27、28隔天早上起床還是只剩22度，整個房間差點變成冷酷，不是我要殘害北極熊，是宿舍冷氣自己在搞ㄚㄚㄚㄚㄚ

**Comment by RogelioKG** [comment]
在家都是開 23 度，直到被室友調教成最低 26 度，現在都是吹 26 度🥴。
但暑假回到高雄可能又要被高雄的天氣給熱死，應該又要調成 23 度。

**Comment by xiang305** [comment]
我都開26或27度，我個人覺得這樣就已經夠冷了。如果真的覺得熱的話再調低也沒差，但我不太理解為什麼有些人就算穿著外套也要開20度左右，這樣把溫度調高不就好了，這樣也比較省電

**Comment by Brownnnn78** [comment]
我自己是都開27度，配上電風扇有時候都會覺得有點冷，為了避免中午太熱但又會浪費電，最近都會想辦法到處去蹭冷氣，就算是盡一點心力希望不要地球暖化吧！晚上現在不開冷氣睡不著，不過我是只要睡著之後就可以關冷氣的那種，所以我都定時1小時左右而已。

**Comment by Aaron0810** [comment]
我們宿舍平常都開25或26，但有時候因為冷氣有點問題他測量的室溫不知道為什麼會降到20甚至更低，感覺是冷媒有點問題。在家的話我早上下午會開到27因為外面都很熱，晚上可能降到26，然後睡前因為我睡覺會怕熱所以會再降到25。

**Comment by Elaine2899** [comment]
我平時都開26-27度，因為我喜歡一邊開著電風扇，因此通常這樣就已經足夠涼了。不過去飯店的話我喜歡開到21.22度，然後蓋超厚超舒服棉被
